packages/backend/README.md

```diff
@@ -125,7 +125,10 @@ ETHEREUM_ALCHEMY_API_KEY=
 # ARBITRUM_TVL_ENABLED=
 # ARBISCAN_API_KEY=
 # ARBITRUM_PROVIDER_URL=
-
+## Enable/disable detailed-tvl endpoint exposure - defaults to true
+# DETAILED_TVL_ENABLED=
+## Should detailed-tvl endpoint return 404 if data is not synced yet - defaults to false
+# SKIP_UNSYNCED_DETAILED_TVL=
```

Comment:
Better name would be `ERROR_ON_UNSYNCED_DETAILED_TV`

Reply:
https://github.com/l2beat/l2beat/pull/1776/commits/c2e508e7976519fcbc105e390e3688848bf87989